### ISSUE TITLE
SteamCache Server partial connection

### ISSUE THREAD

**Issue by robertp92** [steamcache/steamcache-dns#49]
Issue Description:
Partial connection to SteamCache

Docker Run Command:

sudo docker run 
--restart=always
--name steamcache-dns 
-p 192.168.2.10:53:53/udp 
-e STEAMCACHE_IP=192.168.0.44
steamcache/steamcache-dns:latest

Docker Container Output:
 
![steamcache error](https://user-images.githubusercontent.com/9335659/46261092-db907600-c4e6-11e8-8cb3-4134bc847173.png)

Docker Logs:

![docker log](https://user-images.githubusercontent.com/9335659/46261067-93715380-c4e6-11e8-9dec-3618f2ff3ebf.png)

Apologies for the lack of information, I'm fairly new to all this. If there is anything other information I can give you please let me know. 

Basically what is happening is the SteamCache doesn't cache anything from Steam. I keep seeing the above output appearing in yellow. It sometimes appears to post that for a while, but I'm not actually 100% sure what it is caching. If anything at all.

Ran this on the client machine:  
Resolve-DnsName steamcache.cs.steampowered.com

The IP is the same as the SteamCache server when I run that.

Thanks,

Robert
